docs/index.md

```diff
@@ -25,6 +25,7 @@
 * [Adding New Attribute to an Entity](./cookbook/adding-new-attribute-to-an-entity.md)
 * [Adding a New Administration Page](./cookbook/adding-a-new-administration-page.md)
 * [Adding a New Advert Position](./cookbook/adding-a-new-advert-position.md)
+* How to upgrade to new version of Shopsys Framework - please see instructions in [UPGRADE.md file](../UPGRADE.md)
```

Comment:
It is a little bit weird to me that this is in the "cookbook" section. I can probably see your point of view that the `UPGRADE.md` contains step-by-step instructions but still, I would prefer adding the new headline "How to upgrade to new version of Shopsys Framework" and link the upgrade file in the paragraph below the heading